functions/tsconfig.json

```diff
@@ -15,5 +15,5 @@
 		"target": "es2017",
 		"types": ["jest", "node"]
 	},
-	"include": ["src/index.ts"]
+	"include": ["src/**/*"]
```

Comment:
@andrashee, fyi: this fixes the error message in WebStorm

Reply:
but it breaks the tests, it seems🤯

Reply:
The `scripts/` folder seems to be the issue. I moved it out of the `src/` directory now. But we should anyway either refactor that coder or remove it.